resources/dicts/patrols/new_cat.json

```diff
@@ -375,76 +494,72 @@
+        "It's just an odd loner singing to themself. When they see p_l they invite {PRONOUN/p_l/object} to sing too, but instead, p_l offers the somewhat battered looking and strange cat an introduction to c_n. It's got to be far less lonely than their current life.",
+        "s_c walks into an ambush by a group of rogues. It's only by the grace of StarClan that s_c has the fighting skill and confidence to escape, running for {PRONOUN/s_c/poss} life.",
+        "s_c has a bad feeling about this. And maybe {PRONOUN/s_c/subject}{VERB/s_c/'re/'s} being too cautious - but then {PRONOUN/s_c/subject} {VERB/s_c/pick/picks} up on the scent of strange cats around the Twoleg den. Forewarned, s_c retreats to safety."
     ],
     "fail_text": [
-            null,
-            null,
-            "Your patrol walks into an ambush by a group of rogues and everyone is slaughtered.",
-            null
+        null,
+        null,
+        "p_l decides that a quick peek can't hurt. As {PRONOUN/p_l/subject} {VERB/p_l/venture/ventures} into the nest, ears pricked with curiosity, {PRONOUN/p_l/subject} {VERB/p_l/don't/doesn't} notice the lithe forms of rogues moving to block the entrance until it's too late. p_l never makes it back to camp.",
+        null
     ],
     "win_skills": ["excellent fighter"],
     "win_trait": ["careful", "insecure", "nervous"],
     "min_cats": 1,
     "max_cats": 1,
-    "antagonize_text": "r_c can sense it's a trap, and waits just outside, setting their own trap. They manage to set up such a yowl that it sends the rogues running as if LionClan were after them.",
-    "antagonize_fail_text": "r_c can sense its a trap, and tries to get the upper paw, only to find rogues waiting for them there too. r_c escapes with their life, but fear in their heart.",
+    "antagonize_text": "p_l can sense it's a trap, and waits just outside, setting {PRONOUN/p_l/poss} own trap. {PRONOUN/p_l/subject} {VERB/p_l/manage/manages} to set up such a yowl that it sends the rogues running as if LionClan were after them.",
+    "antagonize_fail_text": "p_l can sense it's a trap, and tries to get the upper paw, only to find rogues waiting for {PRONOUN/p_l/object} there too. p_l escapes with {PRONOUN/p_l/poss} life, but fear in {PRONOUN/p_l/poss} heart.",
     "history_text": {
         "reg_death": "This cat died from being ambushed by rogues.",
-        "lead_death": "were killed in a rogue ambush"
+        "lead_death": "{VERB/p_l/were/was} killed in a rogue ambush"
     }
 },
 {
     "patrol_id": "fst_hunt_twolegplacekittens1",
     "biome": "forest",
     "season": "Any",
-    "tags": [
-        "hunting",
-        "no_fail_prey",
-        "new_cat_kittypet_female_dead_litter0_litter1_litter3_abandoned1_abandoned4"
-    ],
+    "tags": ["hunting", "no_fail_prey", "new_cat_kittypet_female_dead_litter0_litter1_litter3_abandoned1_abandoned4"],
```

Comment:
you mentioned wanting these to not be gender specific, the tag you'd want to use would be: "new_cat_kittypet_dead_litter0_litter1_litter3_abandoned1_abandoned4"